strapi/strapi
# fix(i18n): call d&p & i18n migrations in the same file

### What does it do?

Now that i18n is core we can have d&p & i18n migrations together to avoid race conditions when running migrations.

### How to test it?

1. Create a Content Type without draft & publish and i18n.
2. Create some entries
3. Add draft & publish and i18n to the content type
4. It should keep the entries created

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/21269
